src/platform/plugins/shared/discover/public/application/main/components/tabs_view/tabs_view.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { type TabItem, UnifiedTabs, TabStatus } from '@kbn/unified-tabs';
+import React, { useRef, useState } from 'react';
+import { pick } from 'lodash';
+import type { DiscoverSessionViewRef } from '../session_view';
+import { DiscoverSessionView, type DiscoverSessionViewProps } from '../session_view';
+import {
+  createTabItem,
+  internalStateActions,
+  selectAllTabs,
+  selectCurrentTab,
+  useInternalStateDispatch,
+  useInternalStateSelector,
+} from '../../state_management/redux';
+import { useDiscoverServices } from '../../../../hooks/use_discover_services';
+
+export const TabsView = ({ sessionViewProps }: { sessionViewProps: DiscoverSessionViewProps }) => {
```

Comment:
There seem to be an issue with layout when tabs are enabled. "Add field" and the grid navigation are going below the fold because of extra height and scroll. 

<img width="1440" alt="Screenshot 2025-03-27 at 14 35 10" src="https://github.com/user-attachments/assets/cbe9183e-d386-4954-bd42-e015105c119f" />



We could fix it by passing `32` to `kibanaFullBodyHeightCss()` in https://github.com/elastic/kibana/blob/e57d922bf47ff30d20dc788d6222cf566b26148e/src/platform/plugins/shared/discover/public/application/main/components/layout/discover_layout.tsx#L366-L374

Reply:
🤦 I never even noticed... Maybe I should turn on always visible scrollbars. Thanks for the suggestion, I'll look into it for the followup PR.

Reply:
> Maybe I should turn on always visible scrollbars.

Highly recommend! That's one of the first things I do when setting up a new laptop for development 😀